README.md

```diff
@@ -22,6 +22,18 @@ Check the [asdf](https://github.com/asdf-vm/asdf) readme for instructions on how
 
 After using `go get` to install a package you need to run `asdf reshim golang` to get any new shims.
 
+
+### Default `go get` packages
+
+asdf-golang can automatically install a default set of packages with `go get -u $PACKAGE` right after installing a new Go version.
+To enable this feature, provide a \$HOME/.default-golang-pkgs file that lists one package per line, for example:
```

Comment:
I'd consider spelling out `packages` (ie `.default-golang-packages`) to maintain consistency with other asdf plugins. Examples:

https://github.com/asdf-vm/asdf-nodejs#default-npm-packages
https://github.com/danhper/asdf-python#default-python-packages